docs/.stylelintrc.json

```
@@ -0,0 +1,28 @@
+{
+    "extends": ["stylelint-config-standard-scss"],
+    "rules": {
+        "alpha-value-notation": null,
+        "at-rule-empty-line-before": null,
+        "color-function-notation": null,
+        "color-hex-case": null,
+        "custom-property-empty-line-before": null,
+        "custom-property-pattern": null,
+        "declaration-block-no-duplicate-properties": null,
+        "declaration-block-no-redundant-longhand-properties": null,
+        "declaration-block-no-shorthand-property-overrides": null,
+        "hue-degree-notation": null,
+        "indentation": 4,
+        "max-line-length": null,
+        "no-descending-specificity": null,
+        "no-duplicate-selectors": null,
+        "number-leading-zero": null,
+        "number-no-trailing-zeros": null,
+        "property-no-unknown": null,
+        "property-no-vendor-prefix": null,
+        "selector-class-pattern": null,
+        "value-keyword-case": null
+    },
+    "ignoreFiles": [
+        "_site/"
```

```suggestion
        "_site/**"
```

With `_site/`, I'm getting errors in `_site/assets/scss/styles.scss`. I'm guessing this should be a glob that matches files.

---

should we be more specific and add only scss and css to ignoreFiles?

---

> should we be more specific and add only scss and css to ignoreFiles?

`_site` is the build output, so it makes sense to just ignore the whole directory, as originally intended. But it seems that `ignoreFiles` doesn't allow ignoring directories (perhaps that could be done with `.stylelintignore`).